electives/18847.md

```diff
@@ -0,0 +1,41 @@
+# 18-847C: Data Center Computing
+
+| Category | Difficulty |
+|:-:       | :-:        |
+| Reviews  | 5          |
+| Project  | 6          |
+| Present  | 7          |
+
+This class gives a introduction to data center computing issues by reading recently published research papers addressing current problems that datacenters face today, as well as through a semester long research project into a data center problem. 
+
+## Paper Reviews
+
+Each class (meet twice a week), you are required to read and review two reseach papers. During class, these papers are presented and discussed. Sometimes, one of the reseachers who wrote the paper will visit for discussion. The most important part of the reviews is the critical analysis of where they could have improved and your discussion questions. 
```

Comment:
Do you need to prepare your discussion questions beforehand? Are you graded on how much you participate in these discussions?

```suggestion
Each class (meets twice a week), you are required to read and review two research papers. During class, these papers are presented and discussed. Sometimes, one of the researchers who wrote the paper will visit for discussion. The most important part of the reviews is the critical analysis of where they could have improved and your discussion questions. 
```

Reply:
See commit https://github.com/gfieni2022/CMU-ECE-CS-Guide/commit/c2eddb433ab1014933531ef80057d0447d8dc86c